src/caret_analyze/infra/lttng/lttng_info.py

```diff
@@ -237,14 +242,15 @@ def _load_sub_cbs_without_pub(
                 callback_object_intra = int(record_callback_object_intra)
             self._id_to_topic[row['callback_id']] = row['topic_name']
 
+            topic_name = row['topic_name']
             sub_cbs_info[node_id].append(
                 SubscriptionCallbackValueLttng(
                     callback_id=row['callback_id'],
                     node_id=node_id,
                     node_name=node_name,
                     symbol=row['symbol'],
-                    subscribe_topic_name=row['topic_name'],
-                    publish_topic_names=None,
+                    subscribe_topic_name=topic_name,
+                    publish_topics=None,
                     subscription_handle=row['subscription_handle'],
                     callback_object=row['callback_object'],
                     callback_object_intra=callback_object_intra,
```

Comment:
If there is no need for change, it seems better to keep the previous implementation.
```suggestion
            sub_cbs_info[node_id].append(
                SubscriptionCallbackValueLttng(
                    callback_id=row['callback_id'],
                    node_id=node_id,
                    node_name=node_name,
                    symbol=row['symbol'],
                    subscribe_topic_name=row['topic_name'],
                    publish_topics=None,
                    subscription_handle=row['subscription_handle'],
                    callback_object=row['callback_object'],
                    callback_object_intra=callback_object_intra,
```